docs/apb_cli.md

```diff
@@ -156,52 +146,92 @@ be sufficient.
 
 ## Typical Workflows
 
-#### Local Registry
-In order to use the internal OpenShift Docker Registry to source APBs, you must have configured the Ansible Service Broker to use the `local_openshift` type registry adapter. Please see the [config](https://github.com/openshift/ansible-service-broker/blob/master/docs/config.md#local-openshift-registry) section for more information.
+#### Creating and Testing APBs
+In version `2.0.0` and above, `apb` allows you as a developer to develop and test APB images without using an Ansible Service Broker. `apb` allows the user to configure registries to source APBs from and store the list of images locally (in `~/.apb`). Once the images are found, `apb` allows the user to provision the APB directly for testing.
 
-```bash
-apb init my-new-apb
-cd my-new-apb
-apb build
-apb push
-apb list
+Initializing the APB is done with `ansible-galaxy`:
+```
+ansible-galaxy init --type apb <apb-name>
+```
+
+##### Using the internal OpenShift registry
+Modify the APB as you wish. Once complete, we need to create a buildconfig so that an imagestream is populated in a namespace which `apb` can read from. We recommend using the `openshift` namespace for this since by default all imagestreams in `openshift` namespace are accessible to all authenticated users. Regardless, `apb` works with any namespaces. This is documented in the [getting-started document]().
```

Comment:
Suggested change:

Modify the APB as you wish. Once complete, we need to [...]

->

After modifying the APB as desired, we need to [...]

Reply:
to all authenticated users. Regardless, `apb` works with any namespaces.
->
to all authenticated users, but `apb` will work with any accessible namespace.